### PR TITLE
refactor: Entity Fixes + Deferred API Registries

### LINKED ISSUE #731: Bug: Error log when trying to /summon Moa.

### PR DESCRIPTION
**Pull request by bconlon1**
- [x] Closes #731.
- [x] Parachute behavior more accurate to original mod (e.g. rotation and speed, sound and other behavior).
- [x] Switch API registries to DeferredRegister.

### CODE REVIEW

File: src/main/java/com/gildedgames/aether/entity/miscellaneous/Parachute.java [hunk 1]
```diff
@@ -53,45 +50,60 @@ public void tick() {
...
+        float speed = 0.03F;
+        this.moveRelative(speed, vec3);
+        this.move(MoverType.SELF, this.getDeltaMovement());
+        return this.getDeltaMovement();
+    }
+
     public void spawnExplosionParticle() {
-        for (int i = 0; i < 1; ++i) {
+        if (!this.level.isClientSide) {
+            AetherPacketHandler.sendToAll(new ExplosionParticlePacket(this.getId()));
```
Comment (Burning-Cactus) [inline src/main/java/com/gildedgames/aether/entity/miscellaneous/Parachute.java]: This probably doesn't need to be sent to all clients.

File: src/main/java/com/gildedgames/aether/entity/miscellaneous/Parachute.java [hunk 2]
```diff
@@ -53,45 +50,60 @@ public void tick() {
...
+        } else {
             EntityUtil.spawnMovementExplosionParticles(this);
         }
     }
 
     public void die() {
+        for (int i = 0; i < 10; i++) {
```
Comment (Burning-Cactus) [inline src/main/java/com/gildedgames/aether/entity/miscellaneous/Parachute.java]: We should probably stop sending packets in loops, and instead include a number of iterations that the client should go through in a loop so we minimize the number of packets sent.

**Review by Drullkus — DISMISSED** [review]
I don't understand what `DungeonType`s are for. These are redundant since these can already be 1:1'd to the structure system

**Review by Burning-Cactus — CHANGES_REQUESTED** [review]

**Review by Burning-Cactus — APPROVED** [review]
Looks good to me!

**Pull request merged**